sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -1291,9 +1318,14 @@ export class BlobClient extends StorageClient {
         spanOptions
       });
 
+      const wrappedRes = {
+        ...res,
```

Comment:
Weird. The spread object is missing the `_response` properties. 
@jeremymeng @XiaoningLiu @bterlson Any idea why?

Reply:
Is it because of the inheritance? spread operator in an object literal copies **own enumerable properties**

Reply:
We make `_response` non-enumerable, non-writable, and non-configurable at the moment. I discovered this while working on `ServiceClient`.